tests/ledger_sequence_increment.sql

```diff
@@ -12,7 +12,7 @@ with
             , closed_at
             , max(sequence) as max_sequence
         from {{ source('crypto_stellar', 'history_ledgers') }}
-        where (closed_at > current_timestamp - interval 7 day)
+        where TIMESTAMP(closed_at) > TIMESTAMP_SUB('{{ dbt_airflow_macros.ts(timezone=none) }}', INTERVAL 7 DAY )
```

Comment:
nit: No need to cast `closed_at` to timestamp. `closed_at` is already timestamp

Reply:
Addressed in https://github.com/stellar/stellar-dbt-public/pull/79/commits/d453b8a6a36666b5e2a45011b5687796d7e20e90